solarforecastarbiter/datamodel.py

```diff
@@ -960,6 +971,26 @@ def __set_aggregate_normalization__(self):
     object.__setattr__(self, 'normalization', norm)
 
 
+def __set_uncertainty__(self):
+    if isinstance(self.uncertainty, str):
+        try:
+            unc = float(self.uncertainty)
+        except ValueError:
+            # not something we can coerce to a float
+            pass
+        else:
+            object.__setattr__(self, 'uncertainty', unc)
```

Comment:
should this verify between 0 and 100?

Reply:
We don't verify the values of most numeric attributes in datamodel.